### PR TITLE
Adding onnx polish_model

### PR DESCRIPTION
**Pull request by duli2012**
Now we call a function from the official ONNX repo to check/polish the ONNX model generated by the converter.

### CODE REVIEW

File: onnxmltools/convert/main.py [hunk 1]
```diff
@@ -6,33 +6,43 @@
...
 
     from .coreml.convert import convert
-    return convert(model, name, initial_types,
+    onnx_model = convert(model, name, initial_types,
                    doc_string, targeted_onnx, custom_conversion_functions, custom_shape_calculators)
-
+    if targeted_onnx < StrictVersion('1.2'):
+        return onnx_model
+    else:
+        return onnx.utils.polish_model(onnx_model)
```
Comment (wenbingl) [inline onnxmltools/convert/main.py]: Did you try some real model to see if the generated model can really satisfy the onnx checker?

Comment (wschin) [reply onnxmltools/convert/main.py]: We need to test this with all our models!

Comment (duli2012) [reply onnxmltools/convert/main.py]: For SKlearn models, we got onnx checker failer: "No opset import for domain "

Comment (wschin) [reply onnxmltools/convert/main.py]: This means we cannot call this function because ONNX-ML is not supported generally.

**Pull request closed without merging**